zipkin/src/test/java/zipkin/collector/CollectorTest.java

```diff
@@ -13,127 +13,130 @@
  */
 package zipkin.collector;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.logging.Level;
-import java.util.logging.Logger;
+import org.junit.Before;
 import org.junit.Test;
 import zipkin.Span;
 import zipkin.internal.ApplyTimestampAndDuration;
 import zipkin.storage.Callback;
-import zipkin.storage.InMemoryStorage;
+import zipkin.storage.StorageComponent;
 
 import static java.util.Arrays.asList;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
```

Comment:
converted this to mock because a near future change will make this type abstract